### PR TITLE
Conf : merge conf's branch

### PR DESCRIPTION
**Pull request by xavitator**
Provide a module to read a json file. It is corresponded to the issue #11 .

### CODE REVIEW

File: src/core/finder.ml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    end
+  in
+  List.exists exist target
+
+(*
+   Tail-recursive search (and add to [l]) of all files in directories and sub-directories given at second argument. 
+   If the second argument is not a directory, we add it in the list [l].
+*)
+let rec get_sub_files l path : string list =
+  if not (check_directory path) then
```
Comment (maiste) [inline src/core/finder.ml]: Why not merging this function and the one below ? Get_subdir should test if it's a targeted file...

File: tests/test_finder.ml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      let res_finder = List.sort compare ("" :: Finder.get_all_files h) in
+      (check (list string)) ("equal list " ^ (string_of_int turn)) res_cmd res_finder;
+      aux q (turn + 1)
+  in
+  aux test_bank 0
+
+
+(* Test suite for Finder *)
+let suite = [
+  "List finder equality", `Slow, test_finder_eq
```
Comment (maiste) [inline tests/test_finder.ml]: You can use `Quick ;)

**Review by maiste — CHANGES_REQUESTED** [review]

File: tests/test_finder.ml (hunk 2 above)
Comment (xavitator) [reply tests/test_finder.ml]: It is a recursively search of files, and i think that is slow, depended of the number of files.

File: src/core/finder.ml (hunk 1 above)
Comment (xavitator) [reply src/core/finder.ml]: `get_sub_files` takes all subfiles without taking into account the list of regex. Whereas `get_files` takes it into account.
`get_sub_files` is called when a directory is matching (and we take all subfiles).

Comment (maiste) [reply src/core/finder.ml]: There is a problem if we put `*.c` . I know it's annoying to change it now but I wonder if .eosignore is not a better way...

File: tests/test_finder.ml (hunk 2 above)
Comment (maiste) [reply tests/test_finder.ml]: Slow is only for tests you run occasionally according to the documentation.

**Review by maiste — APPROVED** [review]

**Pull request merged**